include/xtensor/xstorage.hpp

```diff
@@ -1169,6 +1192,116 @@ namespace xt
         lhs.swap(rhs);
     }
 
+#if defined(_MSC_VER) && _MSC_VER < 1910
+    #define SELECT_ALIGN XALIGNMENT
+#else
+    #define SELECT_ALIGN (XALIGNMENT != 0 ? XALIGNMENT : alignof(T))
+#endif
+
+    template <class T, std::size_t N, std::size_t Align = SELECT_ALIGN>
+    class alignas(Align) aligned_array : public std::array<T, N>
+    {
+    public:
+        // Note: this is for alignment detection. The allocator serves no other purpose than
+        //       that of a trait here.
+        using allocator_type = std::conditional_t<Align != 0,
+                                                  xsimd::aligned_allocator<T, Align>,
+                                                  std::allocator<T>>;
+    };
+
+    /**
+     * This array class is modeled after ``std::array`` but adds optional alignment through a template
```

Comment:
I guess this comment was related to the previous `array` class and is now related to the `aligned_array` class.

Reply:
fixed that up.